pandas/tests/indexes/datetimes/test_arithmetic.py

```diff
@@ -275,6 +275,20 @@ def test_comparison_tzawareness_compat(self, op):
         with pytest.raises(TypeError):
             op(ts, dz)
 
+    @pytest.mark.parametrize('op', [operator.eq, operator.ne,
```

Comment:
pretty sure we have fixtures for this

Reply:
There's a fixture for the names `["__eq__", ...]`.  This will go on the list of things I'd like to standardize after collecting arithmetic tests in one place.